src/main/java/com/hm/achievement/command/HelpCommand.java

```diff
@@ -101,6 +103,10 @@ public void extractConfigurationParameters() {
 								new String[] { "ACH", "NAME" }, new String[] { "&oach&7", "&oplayer&7" }));
 		langCommandGiveHover = plugin.getPluginLang().getString("aach-command-give-hover",
 				"Player must be online; only Commands achievements can be used.");
+		langCommandAdd = plugin.getChatHeader() + configColor + "/aach add &oamount cat player" + ChatColor.GRAY + " > "
+				+ ChatColor.translateAlternateColorCodes('&', "Increase the progression of an achievement.");
+		langCommandAddHover = plugin.getPluginLang().getString("aach-command-add-hover",
+				"Player must be online; All achievements can be used except Commands achievements; Write a subcategory like this: category.subcategory");
```

Comment:
This is too long as well. Recent Minecraft versions handle multi-line hovers, but this isn't the case on older ones. I just tried launching version 1.8.4, with my screen resolution I can only see `Write a subcategory like this: category.subcategory`, the rest is hidden! 😄 

A simple `Player must be online; mainly used for Customs achievements.` may do the trick. We can add more details to the Wiki.